Initium-Odp/src/com/universeprojects/miniup/server/services/PropertiesService.java

```diff
@@ -80,4 +80,18 @@ public void buyHouse(ODPDBAccess db, CachedDatastoreService ds, CachedEntity use
 		}
 		
 	}
+	
+	public void rediscoverHouses(Key userKey)
```

Comment:
I just migrated the discoverAllPropertiesFor() method into the ODP. This is what you're looking for.